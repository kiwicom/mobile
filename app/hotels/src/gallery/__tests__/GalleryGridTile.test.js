// @flow

import * as React from 'react';
import renderer from 'react-test-renderer';

import Tile from '../GalleryGridTile';

const voidCallback = () => {};

it('renders with marginRight if not last in the row', () => {
  expect(
    renderer
      .create(
        <Tile
          width={10}
          lastInRow={false}
          imageUrl="http://.../"
          gap={5}
          imageIndex={1}
          onTilePress={voidCallback}
        />,
      )
      .toJSON(),
  ).toMatchSnapshot();
});

it('renders without marginRight if last in the row', () => {
  expect(
    renderer
      .create(
        <Tile
          width={10}
          lastInRow={true}
          imageUrl="http://.../"
          gap={5}
          imageIndex={1}
          onTilePress={voidCallback}
        />,
      )
      .toJSON(),
  ).toMatchSnapshot();
});
