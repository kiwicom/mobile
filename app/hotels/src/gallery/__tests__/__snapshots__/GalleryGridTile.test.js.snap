// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with marginRight if not last in the row 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
>
  <Connect(NetworkImage)
    resizeMode="cover"
    source={
      Object {
        "uri": "http://.../",
      }
    }
    style={
      Object {
        "height": 10,
        "marginBottom": 2,
        "marginRight": 5,
        "width": 10,
      }
    }
  />
</TouchableOpacity>
`;

exports[`renders without marginRight if last in the row 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
>
  <Connect(NetworkImage)
    resizeMode="cover"
    source={
      Object {
        "uri": "http://.../",
      }
    }
    style={
      Object {
        "height": 10,
        "marginBottom": 2,
        "width": 10,
      }
    }
  />
</TouchableOpacity>
`;
